PriRa.GitHub.Actions.Ftp/Options.cs

```diff
@@ -62,5 +62,10 @@ public Options()
                 "Folders to be ignored in both source and destination, separated by a pipe (|) character.")]
         public string SkipDirectories { get; set; }
 
+        [Option("workingDirectory",
+            Required = false,
+            Default = "httpdocs",
```

Comment:
The use of this default introduces a breaking change.

Reply:
I slightly disagree. It is only an extra property which is not mandatory. On lots of systems this is not considered breaking.

If you still consider this as breaking, you can upgrade the version number, so people can take appropriate action when upgrading.